posts/2022-09-29-22.0.0.11-beta.adoc

```diff
@@ -0,0 +1,174 @@
+[#support]
+==  Jakarta EE 10: Support Jakarta Servlet 6.0
+
+The Open Liberty servlet-6.0 feature is an implementation of the Jakarta EE 10 Servlet 6.0 Specification. It includes a number of new features, specification clarifications as well as deprecation features.
+
+Prior to servlet-6.0, there were only a few specific methods that could be set for an application's cookie attributes. There was not an easy way for application to set a new attribute (for example, the recent browser security's SameSite attribute). 
+
+One of the new features in servlet-6.0 is the ability to set any attribute using a Cookie's setAttribute method. In addition, servlet-6.0 also introduces some new request APIs to aid the application's debugging. Application developer can track or refer to a request using the ServletRequest's getRequestId or obtain details of the network connection that is being used by the request by calling ServletRequest's getServletConnection.
+
+You can activate the features incorperating the following in your `server.xml`:
+
+[source, xml]
+----
+   <featureManager>
+       <feature>servlet-6.0</feature>
+   </featureManager>
+----
+
+The Jakarta Servlet 6.0 Specification and Javadocs can be found link:https://jakarta.ee/specifications/servlet/6.0/[here].
+
+
+// https://github.com/OpenLiberty/open-liberty/issues/22115
+[#connectors]
+== Connectors-2.1
+
+This feature allows Open Liberty users that want to use the latest release of the link:https://github.com/eclipse-ee4j/jca-api[Jakarta resource specification] within their application.
+
+The main update to connectors to come out of this feature is the addition of Java generics support to the jakarta.resource.cci.ResourceFactory, jakarta.resource.cci.IndexedRecord, and jakarta.resource.cci.MappedRecord classes 
+
+Open Liberty is keeping our connectors implementation up to date with the latest specification.
+
+You can activate the features incorperating the following in your `server.xml`:
+[source, xml]
+----
+<server>
+ <featureManager>
+   <feature>connectors-2.1</feature>
+ </featureManager>
+</server>
+----
+
+All other `resourceAdapter` configurations within server.xml can remain unchanged for current applications.
+
+You can find out more about this specific update below:
+
+* link:https://github.com/eclipse-ee4j/jca-api[Spec Project]
+* link:https://mvnrepository.com/artifact/jakarta.resource/jakarta.resource-api[Spec API]
+* link:https://jakarta.ee/specifications/connectors/2.1/apidocs/jakarta.resource/module-summary.html[Spec Javadoc]
+
+
+// https://github.com/OpenLiberty/open-liberty/issues/22078
+[#persistence]
+== Persistence 3.1
+
+Jakarta Persistence API is a richly featured API which provides object-model approach to persisting, fetching, and modifying data stored on a Relational Database System.
+
+JPA has been around since EE 5, but before JPA you had to use EJB's Bean or Container managed entities, or raw JDBC to interact with database storage.
```

Comment:
```suggestion
JPA has been around since Java EE 5.  To interact with database storage before JPA, you had to use EJB bean or container managed entities, or raw JDBC.
```